hazelcast/src/main/java/com/hazelcast/concurrent/lock/LockEvictionProcessor.java

```diff
@@ -70,16 +69,12 @@ private void sendUnlockOperation(Data key, int version) {
     private void submit(UnlockOperation operation, Data key) {
         int partitionId = nodeEngine.getPartitionService().getPartitionId(key);
         OperationService operationService = nodeEngine.getOperationService();
-        operation.setNodeEngine(nodeEngine);
-        operation.setServiceName(SERVICE_NAME);
         operation.setPartitionId(partitionId);
-        OperationAccessor.setCallerAddress(operation, nodeEngine.getThisAddress());
-        operation.setCallerUuid(nodeEngine.getLocalMember().getUuid());
         operation.setOperationResponseHandler(unlockResponseHandler);
         operation.setValidateTarget(false);
         operation.setAsyncBackup(true);
 
-        operationService.execute(operation);
+        operationService.invokeOnTarget(SERVICE_NAME, operation, nodeEngine.getThisAddress());
     }
```

Comment:
Looks good. The other way of doing it is having a callback with execute. The invokeOnTarget can be back-pressured. But it's fine.